### ISSUE TITLE
Support tabs

### ISSUE THREAD

**Issue by egoist** [egoist/devdocs-app#9]
<img width="553" alt="2017-02-01 9 42 53" src="https://cloud.githubusercontent.com/assets/8784712/22509047/640ec3ae-e8c7-11e6-9514-7023193914af.png">

the links in the screenshot will open a new tab in default browser, I think instead it should open it in a new app window or a new tab inside the app

**Comment by alexandr-san4ez** [comment]
I think it should be opened in a single tab.
This is a personal opinion. :)

**Comment by gluons** [comment]
In my opinion, tab feature is very useful.
When I'm working with multiple library/framework, I don't want to search across framework frequently.
1 framework per 1 tab is really useful.

**Comment by ELLIOTTCABLE** [comment]
Yeah, tabs is a :+1: here! I don't think it needs complicated support for framework-per-tab or whatnot, personally — just duplicate the browser behaviour:

* **⌘-T** to create a new tab at the DevDocs home-screen,
* **⌘-click** a link to open a particular link in a new tab,
* **⌘-W** to close the current tab.

(Or whatever their Windows equivalents are.)

**Comment by egoist** [comment]
Oh, I found the reason why these links would be opened in your default browser: **the doc for that isn't enabled 😅**

**Comment by lloeki** [comment]
- Multiple tabs per window are good. Use case: I frequently want multiple pages of the same framework open (e.g Go's `net/mail`, `mime` and `mime/multipart` pages) because I keep going from one to the other.
- Multiple windows are good. Use case: I use multiple desktops, each one as a workspace for a given task. In each one I want to have documentation for the task I'm working on. Currently this is not possible.

**Comment by Spy-Seth** [comment]
As a long time user, I would not see this king of behavior in the application. As an user the simplicity of `devdocs-desktop` is really important for me. I would not transform the application into a another internet browser: this is not the job of this application.

**Comment by ELLIOTTCABLE** [comment]
@Spy-Seth but ”tabs” do not “another internet browser” make! It's one, relatively small, usability feature — that browsers *also* happen to have.

We're not asking for a Firefox-esque tab tree! 🤣

**Comment by egoist** [comment]
Yeah we can have a minimal design as well, if you don't create a new tab it's no different from the current design.
